### ISSUE TITLE
[Question] Does a Task count toward an Activity's Goal?

### ISSUE THREAD

**Issue by Mlk1225** [Medvedev91/timeto.me#94]
- Description: The problem appears in my workflow and interface interactions below:
    - A **Goal** has been set for an **Activity**
    - Select a **Task** and prepare to execute it -> App prompts to select an **Activity** for this **Task** -> select a specific **Activity** type (e.g. *Reading*) -> Finish executing the **Task**
    - Problem arise: The time spent executing the **Task** is not counted in the **Goal** of that **Activity**
- I wonder if there is something *wrong* with the way I operate, or if there are some details that have been *ignored*?

- In addition, I guess the design of **Goal** is to set a **Task** as one of the Goals of the **Activity**? I can understand this implementation and guess that this difference may stem from the understanding of **Goal**:

|          | original design (maybe)                                                                                                                                              | my understanding                                                                                                                                                                         |
| -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Design   | The **Goal** is another form of **Task**.<br><br>Treat the Goal as a specific Task, and when the user needs to execute a Task, they can directly click related Goal. | The **Goal** is for the **Activity**.<br><br>Each Task is collected into different activities, and the Goal is used as a standard to judge whether the day's behavior is up to standard. |
| Drawback | Goal will be less inflexible and inconvenient in terms of arranging time?                                                                                            | If a Task is linked to a Goal, then multiple Goals under the same Activity are meaningless or overlapped?                                                                                |

- To sum up, I really like the design of this app, which really helps me a lot. Ask this question here to know how the **Goal** designed and also hope to refine my workflow. :)

**Comment by Medvedev91** [comment]
Hi,

Thanks for the detailed explanation. Is it similar to https://github.com/Medvedev91/timeto.me/issues/85 ?

**Comment by Mlk1225** [comment]
Oh, yes, sorry I didn't double-check beforehand and took up your time.

I try it like #85 and find that work when setting goal's note blank.

The whole design is awesome, maybe you can consider modifying the existing guidance "Note (optional)" that appears when setting a new goal as it is confusing and fails to show the whole function, or mentioning this feature more precisely in _How to Use the App_.

Thank you again for your work!

**Comment by Medvedev91** [comment]
Yes, I will do smth. A lot of people are struggling with this.
